addon/components/frost-select-dropdown.js

```diff
@@ -372,6 +410,9 @@ export default Component.extend({
     const clonedDropdownListElement = dropdownListElement.cloneNode(true)
     const clonedTextElements = clonedDropdownListElement.querySelectorAll('.frost-select-list-item-text')
     const textElements = dropdownListElement.querySelectorAll('.frost-select-list-item-text')
+    const clonedSecondaryTextElements = clonedDropdownListElement
+                                        .querySelectorAll('.frost-select-list-secondary-item-text')
```

Comment:
This mega tabbing isn't how we usually do it. Instead I'd go with:
```
const clonedSecondaryTextElements = clonedDropdownListElement
  .querySelectorAll('.frost-select-list-secondary-item-text')
```
Otherwise if anyone ever renames the variable in line 413 they'll also need to adjust the spacing in line 414.